src/postgres/contrib/yb_auh/yb_auh.c

```diff
@@ -47,8 +47,8 @@ typedef struct ybauhEntry {
   long request_id;
   uint32 wait_event;
   char wait_event_aux[16];
-  char top_level_node_id[16];
-  char client_node_ip[16];
+  uint64_t top_level_node_id[2];
```

Comment:
not a big fan of having uint64_t arrays.

conceptually find it simpler to use char/uint8 [] arrays of size 8/16.

can always get uint64_t foo = *(static_cast<uint64_t*>&bar[0]) to read the array as a unit64 number.

Reply:
using uint8 arrays will make the code longer in places like this. Is this okay?

```
    bg_wait_states_.push_back(std::make_shared<yb::util::WaitStateInfo>(
                                yb::util::AUHMetadata{
                                    .top_level_node_id = {0, 1},
                                    .top_level_request_id = {0, 2},
                                    .query_id = -2}));
```

Reply:
it's ok for now.But I think it is cleaner to just track as bytes/chars.

We could make these constants in some file. But let's get to it later.